### PR TITLE
sbt forking jvm

### PR DESCRIPTION
**Pull request by aweng98**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by aweng98** [comment]
jenkins test pass

https://fc-jenkins.dsp-techops.broadinstitute.org/job/leonardo-fiab-test-runner/2364/artifact/test-reports/

**Comment by aweng98** [comment]
pass https://fc-jenkins.dsp-techops.broadinstitute.org/view/Testing/view/Pipeline%20Jobs/job/swatomation-pipeline/5497/

jenkins retest

File: automation/project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version = 0.13.15
\ No newline at end of file
+sbt.version = 1.1.4
```
Comment (aweng98) [inline automation/project/build.properties]: this is the latest sbt version. leo dev project version is at older version.

File: automation/src/test/resources/logback-test.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aweng98) [inline automation/src/test/resources/logback-test.xml]: should this config file be log4j or logback?

**Comment by aweng98** [comment]
This works on localhost regardless headless is true or false. also, no longer need to set VM argument when invoking sbt to run tests locally. this will do. `sbt "testOnly *PublishSpec"`

File: automation/src/test/resources/logback-test.xml (hunk 2 above)
Comment (kyuksel) [reply automation/src/test/resources/logback-test.xml]: For what it's worth, the last time I checked, logback was the long-term maintained solution.

File: automation/src/test/resources/logback-test.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        <append>false</append>
+        <immediateFlush>true</immediateFlush>
+    </appender>
+
+    <root level="DEBUG">
+        <appender-ref ref="CONSOLE"/>
+        <appender-ref ref="FILE" />
+    </root>
+
+</configuration>
```
Comment (kyuksel) [inline automation/src/test/resources/logback-test.xml]: add new line

**Pull request merged**